content/metadata.yaml

```diff
@@ -525,9 +525,9 @@ author_info:
   #   affiliations:
   #   - Boise State University
   # - name: Trang Le
-  #   orcid: null
-  #   github: null
-  #   twitter: null
+  #   orcid: 0000-0003-3737-6565
```

Comment:
You must uncomment your fields in metadata.yaml

Reply:
Done in [this commit](https://github.com/dhimmel/psb-manuscript/pull/1/commits/1c1e41c01735b9d7a46a7a7311236821b922bcb5).